src/multicloud-connector/azext_multicloud_connector/aaz/latest/arc_multicloud/public_cloud_connector/_delete.py

```diff
@@ -19,7 +19,7 @@ class Delete(AAZCommand):
     """Delete a PublicCloudConnector
 
     :example: PublicCloudConnectors_Delete
-        az arc-multicloud public-cloud-connector delete --resource-group rgpublicCloud --name skcfyjvflkhibdywjay
+        az arc-multicloud public-cloud-connector delete --resource-group rgpublicCloud --name myConnector
```

Comment:
rgpublicCloud -> multiCloudRG (Lets use multicloud everywhere instead of publicCloud)
myConnector -> awsConnector

Reply:
Please take this as global comment.